### PR TITLE
npm CLI: Added Categories

### PR DESCRIPTION
**Pull request by JelaniThompson**
Hey there, I _think_ I have an understanding of what you'd like here. Is this okay?

## Description of new Instant Answer, or changes
I added the categories listed on the sidebar here: [https://docs.npmjs.com/cli/whoami](https://docs.npmjs.com/cli/whoami). Just wanted to make sure I'm not missing anything!

## Related Issues and Discussions
Related links: [https://github.com/duckduckgo/zeroclickinfo-fathead/issues/747](https://github.com/duckduckgo/zeroclickinfo-fathead/issues/747)

## People to notify
@pjhampton

---

Instant Answer Page:  https://duck.co/ia/view/npm_cli

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by pjhampton** [comment]
I think this file should be called `keywords.txt`

File: cover/npm-cli.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+access
```
Comment (moollaza) [inline cover/npm-cli.txt]: @JelaniThompson please remove this file. The Keywords.txt is the only file you need 👍

**Comment by JelaniThompson** [comment]
Done!

**Comment by moollaza** [comment]
@JelaniThompson Thanks! This LGTM :+1:

**Comment by moollaza** [comment]
@JelaniThompson the `/cover` directory is in the wrong place. It needs to be in `lib/fathead/npm_cli` -- I didn't notice it was incorrectly placed.

Please open a new PR and move it to the correct location 👍

**Comment by pjhampton** [comment]
https://github.com/duckduckgo/zeroclickinfo-fathead/pull/821

**Pull request merged**